filer/private/sass/components/_navigator.scss

```diff
@@ -706,115 +682,3 @@ body {
     display: table-cell;
     vertical-align: middle;
 }
-
-.filebrowser .navigator-thumbnail-list {
```

Comment:
I just reverted to complete pull request, ie. #1255.
It probably is easier for whomever works on it again, to integrate it into the then master branch.
The CSS here is only used for that purpose, so keeping it may introduce a conflict in the future.